<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tintalle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="connection_groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>295</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Connection</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="saber_select_box"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_label">
           <property name="text">
            <string>Searching...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="connect_button">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="content_tabWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="color_tab">
         <attribute name="title">
          <string>Color</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="2,1" columnstretch="1,0">
          <item row="0" column="0">
           <widget class="QGroupBox" name="color_groupbox">
            <property name="title">
             <string>Color Select</string>
            </property>
            <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0,1">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item row="1" column="2">
              <widget class="QSlider" name="g_slider">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksAbove</enum>
               </property>
               <property name="tickInterval">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>White</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="w_spinbox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QSlider" name="b_slider">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksAbove</enum>
               </property>
               <property name="tickInterval">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Blue</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Green</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QSlider" name="w_slider">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksAbove</enum>
               </property>
               <property name="tickInterval">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="g_spinbox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSlider" name="r_slider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksAbove</enum>
               </property>
               <property name="tickInterval">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="r_spinbox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>75</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="b_spinbox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Red</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="controls_groupbox">
            <property name="title">
             <string>Controls</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QComboBox" name="color_bank_select_box">
               <property name="placeholderText">
                <string>Color Bank</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="reset_color_changes_button">
               <property name="text">
                <string>Reset Changes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="color_save_button">
               <property name="text">
                <string>Save Bank to Saber</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="preview_color_button">
               <property name="text">
                <string>Preview Color on Saber</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="effect_groupbox">
            <property name="title">
             <string>Effect Select</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1">
             <item row="0" column="0">
              <widget class="QRadioButton" name="main_radioButton">
               <property name="text">
                <string>Main</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="clash_radioButton">
               <property name="text">
                <string>Clash</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QRadioButton" name="swing_radioButton">
               <property name="text">
                <string>Swing</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="main_color_label">
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="clash_color_label">
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="swing_color_label">
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="sound_tab">
         <attribute name="title">
          <string>Sound</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QGroupBox" name="effects_groupbox">
            <property name="title">
             <string>Effects</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="poweron_checkBox">
               <property name="text">
                <string>Power On</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">effects_buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="poweroff_checkBox">
               <property name="text">
                <string>Power Off</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">effects_buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="hum_checkBox">
               <property name="text">
                <string>Hum</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">effects_buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="clash_checkBox">
               <property name="text">
                <string>Clash</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">effects_buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="swing_checkBox">
               <property name="text">
                <string>Swing</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">effects_buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="smoothswingA_checkBox">
               <property name="text">
                <string>SmoothSwing A</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">effects_buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="smoothswingB_checkBox">
               <property name="text">
                <string>SmoothSwing B</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">effects_buttonGroup</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QGroupBox" name="files_groupbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Files</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="reset_sound_changes_button">
               <property name="text">
                <string>Reset Changes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sound_save_button">
               <property name="text">
                <string>Save Changes</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="erase_button">
               <property name="text">
                <string>Erase Sounds</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="upload_button">
               <property name="text">
                <string>Upload Files</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="freespace_label">
              <property name="text">
               <string>Free Space: --- MB</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="usedspace_label">
              <property name="text">
               <string>Used Space: --- MB</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="totalspace_label">
              <property name="text">
               <string>Total Space: --- MB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QTreeWidget" name="files_treeWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>File Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Size</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEditLogger" name="logTextBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>42</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="action_Refresh_Ports"/>
    <addaction name="action_Reload_Config"/>
   </widget>
   <widget class="QMenu" name="menuTroubleshooting">
    <property name="title">
     <string>Troubleshooting</string>
    </property>
    <addaction name="action_Show_Hide_Log"/>
    <addaction name="action_Debug_Mode"/>
   </widget>
   <widget class="QMenu" name="menuFirmware">
    <property name="title">
     <string>Firmware</string>
    </property>
    <addaction name="action_Check_for_Latest_Firwmare"/>
    <addaction name="action_Install_Firmware_from_File"/>
   </widget>
   <addaction name="menuConnection"/>
   <addaction name="menuFirmware"/>
   <addaction name="menuTroubleshooting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Refresh_Ports">
   <property name="text">
    <string>&amp;Refresh Ports</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Show_Hide_Log">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show/Hide &amp;Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Debug_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Debug Mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Reload_Config">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reload &amp;Config</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Check_for_Latest_Firwmare">
   <property name="text">
    <string>&amp;Check for Latest Firwmare</string>
   </property>
  </action>
  <action name="action_Install_Firmware_from_File">
   <property name="text">
    <string>&amp;Install Firmware from File...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextEditLogger</class>
   <extends>QPlainTextEdit</extends>
   <header>qtexteditlogger</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="effects_buttonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
